### ISSUE TITLE
[Bug]: qbit_manage resumes torrents

### ISSUE THREAD

**Issue by grapexy** [StuffAnThings/qbit_manage#175]
### Describe the Bug

After untagging torrents that were previously marked with noHL, qbit_manage resumes pausedUP torrents.
https://github.com/StuffAnThings/qbit_manage/blob/f50d68d59607e35b2f4025e54b20fd85e51be5fb/modules/qbittorrent.py#L371-L374

That's rather quirky side-effect and at the very least, should be off by default and be enabled with a flag explicitly. There's no reason to mess with torrent states like that.

### Logs

no need

### Installation

Docker

### Version Number

3.3.0

### What branch are you on?

master

**Comment by bobokun** [comment]
You can try it yourself too. Take a torrent that is already paused due to it meeting the share ratio limits. Change the limits to not use any share limits and it will not automatically resume.

**Comment by grapexy** [comment]
Right, but that is not the responsibility of noHL tagging. noHL tag job should be doing preferably one thing and one thing only - tagging and untagging torrents with or without hardlinks. There is no logical link between hardlinks and share limits, torrent state or any other functionality of `qbit_manage`. Force chaining them together constrains this functionality to only one use case, completely prevents any other use case outside of the presumed and causes surprises.

For example, my use case consists of seeding torrents on seedbox until their ratio requirements are satisfied, and keeping locally cloned torrent in paused state until seedbox torrent is no longer there, at which point local torrent is resumed. `qbit_manage` intervenes in this process and may even lead to bans on some trackers.

I can think of many other use cases apart from mine where this side-effect would cause unexpected surprises. If the torrent was paused manually, there really shouldn't be any process that resumes it unless explicitly requested.

**Comment by bobokun** [comment]
yeah, that makes sense. I'll add a new config option and have it default to false

**Comment by bakerboy448** [comment]
Id suggest - to not break existing installs

- new installs default to false
- Existing installs if no hard links is true default to true

**Comment by bobokun** [comment]
There isn't really a way for me to determine existing installs or not. Here's my suggestion..

I'll leave it default to true if you're running it for the first time and it's not in your config.yml, however in the sample config I'll set it as false so those that are new would be copying a sample config with it set to false.

**Comment by bakerboy448** [comment]
Think that's a happy medium.

Wasn't sure if you were able to write new config values conditionally based on others or not

**Comment by grapexy** [comment]
> I'll leave it default to true if you're running it for the first time and it's not in your config.yml, however in the sample config I'll set it as false so those that are new would be copying a sample config with it set to false.

This sounds fine to me. However, I'd suggest throwing in a deprecation warning and switching the default to False within couple versions down the line.

**Comment by bakerboy448** [comment]
Why would it need to need to do that?

- new config template will be false
- existing config template without that item will be defaulted to true to match existing behavior.
- Users can set to false if/as needed

**Comment by bobokun** [comment]
this is now added in the latest develop branch.

https://github.com/StuffAnThings/qbit_manage/blob/e86c7757766a8d3eb71bb47d216cc3729df56aee/config/config.yml.sample#L154-L155
